apps/trend-rights/components/WhyWeAre.tsx

```diff
@@ -1,20 +1,21 @@
-import { FC, useEffect, useMemo, useRef } from 'react'
+import { FC, useMemo } from 'react'
 
 import {
+  Box,
   Button,
+  Flex,
   Image,
   SimpleGrid,
   Stack,
   Text,
-  Box,
-  Flex,
 } from '@chakra-ui/react'
 import { useRouter } from 'next/router'
 import { useTranslation } from 'next-i18next'
 import { NextSeoProps } from 'next-seo'
 
+import { BLOG_CATEGORIES } from '@fc/config'
 import { Blog } from '@fc/types'
-import { Hero, AnimatedBox, BlogCard, Container } from '@fc/ui'
+import { AnimatedBox, BlogCard, Container, Hero } from '@fc/ui'
 
 export type WhyWeAreProps = {
```

Comment:
Can you remove `export`

```suggestion
type WhyWeAreProps = {
```